localization/en-us/.gitignore

```diff
@@ -0,0 +1 @@
+*.json
```

Comment:
Why did we add these to the git ignores?

Reply:
Just added this gitignore so we would't commit the translated files by accident when messing around with the script. Also PERSONALLY I think that this keeps the folder created for us on git, so we shouldn't fall on some edge cases when we have to account for cache and checking if the file is already there or not in the script, I had some problems with this in the past, but is totally optional.